browser/sources.gni

```diff
@@ -647,3 +648,7 @@ if (!is_android) {
   brave_chrome_browser_allow_circular_includes_from +=
       [ "//brave/browser/ui/webui/brave_new_tab_page_refresh" ]
 }
+
+if (target_os == "mac" && enable_updater) {
```

Comment:
I don't think we need `enable_updater` since `//brave/browser:mac_features` has only `is_mac` and we might extend mac_features with non update related features in the future

Reply:
it's a bit odd to me that this is called `mac_features` instead of of being in something like `brave/browser/updater/features.h`

Reply:
I removed `enable_updater` in 3a4ce2d923439a5541d6d2ed78f2387c53780053.

It's in `mac_features` because we currently only offer Omaha 4 (via the feature flag referenced in this PR) on macOS. Once we offer it on other OSs (most importantly Windows), we can move it to a platform-agnostic place.

Reply:
I think it would be better to have it in a place that made more sense for the feature in general. We have a lot of features that are platform specific and none of them are in platform_features so I think it's better to stick with conventions here. It's not unusual to have features that are initially scoped to one platform and later expand to others.

Reply:
I'm fine doing it in a follow-up since it wasn't added in this PR, but I don't want other people to see this and think it's a good place for flags that are only used on mac right now.

Reply:
Thank you for adding this; I'll do it in a follow-up PR.